client/gutenboarding/index.ts

```diff
@@ -0,0 +1,14 @@
+/**
```

Comment:
This file might as well be `.js` since there's no TypeScript here.

Reply:
On the contrary, this file written entirely in TypeScript 🙂 

I'd like to see more and more `.ts` and `.tsx` extensions. Effectively, it turns on type checking. Simple files like this are an easy place to start with clean, well-typed code. Using the TypeScript extensions encourages us to keep things that way.